includes/Globals.php

```diff
@@ -59,5 +59,10 @@
 $postTitle = 'PostTitle';
 $assignedUser = 'AssignedUser';
 
+// admin settings
+$settings = 'AdminSettings';
+	$setKey = 'setting';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->